java/src/main/java/ai/rapids/cudf/CuFile.java

```diff
@@ -56,6 +56,7 @@ private static synchronized void initialize() {
         initialized = true;
       } catch (Throwable t) {
         log.error("Could not load cuFile jni library...", t);
```

Comment:
This is a log-then-throw antipattern, we should just throw and let the catching code decide whether to log.  This is very likely to be doubly-logged in practice.

Reply:
Done.